registryindexer/main.jl

```diff
@@ -13,26 +13,26 @@ Pkg.instantiate()
 
 using ProgressMeter, Query, JSON, UUIDs, Tar, CodecZlib
 
-function get_all_package_versions(;max_versions=typemax(Int))
+function get_all_package_versions(; max_versions = typemax(Int))
     registry_folder_path = joinpath(homedir(), ".julia", "registries", "General")
     registry_path = joinpath(registry_folder_path, "Registry.toml")
 
     registry_content = Pkg.TOML.parsefile(registry_path)
 
     packages = registry_content["packages"] |>
-        @map({
-            name = _[2]["name"],
-            uuid = UUID(_[1]),
-            path = _[2]["path"]
-        }) |>
-        @mutate(
-            versions = (Pkg.TOML.parsefile(joinpath(registry_folder_path, _.path, "Versions.toml")) |>
-                @map(i->{version=VersionNumber(i[1]), treehash=i[2]["git-tree-sha1"]}) |> 
-                @orderby_descending(i->i.version) |>
-                @take(max_versions) |>
-                collect)
-        ) |>
-        collect
+               @map({
+                   name = _[2]["name"],
+                   uuid = UUID(_[1]),
+                   path = _[2]["path"]
+               }) |>
+               @mutate(
+                   versions = (Pkg.TOML.parsefile(joinpath(registry_folder_path, _.path, "Versions.toml")) |>
+                               @map(i -> {version = VersionNumber(i[1]), treehash = i[2]["git-tree-sha1"]}) |>
+                               @orderby_descending(i -> i.version) |>
+                               @take(max_versions) |>
+                               collect)
+               ) |>
+               collect
```

Comment:
The indentation here seems to fall back to the align defaults, whereas it should just be based on indent levels.

Reply:
weird, this looks like a bug